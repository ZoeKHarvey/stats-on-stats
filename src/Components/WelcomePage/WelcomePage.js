import React from 'react';
import fullrink from '../../images/full-rink.jpeg'
import './WelcomePage.scss';
import ice from '../../images/ice2.jpg';
import NavLogos from '../NavLogos/NavLogos'

export const WelcomePage = ({getSingleRoster}) => {
    return(
      <section className="section-welcome">
        <h1 className="welcome__h1--title">ICED</h1>
        <img className="section__img--stadium" src={fullrink} alt="nhl stadium" />
        <img className="section__img--paint" src={ice} alt="paint backround" />
        <NavLogos getSingleRoster={getSingleRoster} /> 
      </section>
    );
  };

export default WelcomePage;