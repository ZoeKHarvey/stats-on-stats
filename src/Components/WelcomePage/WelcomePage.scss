.section__img--stadium {
  position: absolute;
  z-index: -5;
  width: 48%;
  margin-left: 2%;
  margin-top: 3%;
  opacity: .5
}

.section__img--paint {
  margin-right: 34%;
  position: absolute;
  z-index: -10;
  width: 100%;
  height: 100vh;
  margin-top: 0;

}

.section-welcome {
  height: 92vh;
}

.welcome__h1--title {
  font-family: "Anton", sans-serif;
  position: absolute;
  font-size: 5em;
  margin-left: 15%;
  text-shadow: 2px 2px #CCDBDC;
}