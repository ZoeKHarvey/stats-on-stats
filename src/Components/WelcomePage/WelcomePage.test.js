import React from 'react';
import { shallow } from 'enzyme'
import { WelcomePage } from './WelcomePage';

describe('WelcomePage', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(
      <WelcomePage />
    );
  });

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
