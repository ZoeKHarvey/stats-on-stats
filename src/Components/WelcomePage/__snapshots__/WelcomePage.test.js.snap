// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WelcomePage should match the snapshot 1`] = `
<section
  className="section-welcome"
>
  <h1
    className="welcome__h1--title"
  >
    ICED
  </h1>
  <img
    alt="nhl stadium"
    className="section__img--stadium"
    src="full-rink.jpeg"
  />
  <img
    alt="paint backround"
    className="section__img--paint"
    src="ice2.jpg"
  />
  <NavLogo />
</section>
`;
